testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -66,11 +68,14 @@ def CRD_iter(fn):
         for ref, other in zip(CRD_iter(CRD), CRD_iter(outfile)):
             assert ref == other
 
-    def test_write_EXT(self):
-        # TODO: Write tests that use EXT output format
-        # Must have *lots* of atoms, maybe fake the system
-        # to make tests faster
-        pass
+    def test_write_EXT(self, u, outfile):
+        # Use the `extended` keyword to force the EXT format
+        u.atoms.write(outfile, extended=True)
+
+        with open(outfile, 'r') as inf:
+            format_line = inf.readlines()[2]
+            assert 'EXT' in format_line, "EXT format expected"
+            
```

Comment:
Add a test `test_write_EXT_read`  that MDAnalysis can *read* the file, too.

Reply:
I included a test to assert: 1) number of atoms, 2) number of residues, 3) number of segments and 4) center of geometry. 

Since the EXT format supports more decimals for coordinates than the standard noextended format, I used `np.around(cog,6)` for the assertion of center of geometries.